lib/mongoid/association/embedded/embeds_many/proxy.rb

```diff
@@ -107,12 +107,25 @@ def clear
           #
           # Use #size if you want the total number of documents.
           #
+          # If args or block are present, #count will delegate to the target and
```

Comment:
#count will delegate to the #count method on +target+ ?

Reply:
Done.